src/java/src/main/java/triton/client/InferResult.java

```diff
@@ -269,6 +269,50 @@ public double[] getOutputAsDouble(String output) {
         return (double[])getOutputImpl(out, double.class, ByteBuffer::getDouble);
     }
 
+    public float[] getOutputFp16AsFloat(String output) {
```

Comment:
Could you add the comments for this function like the other ones just for the consistency?